src/ci/bin/test.inc.sh

```diff
@@ -532,7 +532,14 @@ cromwell::private::setup_prior_version_resources() {
         grep 'val cromwellVersion' "${CROMWELL_BUILD_ROOT_DIRECTORY}/project/Version.scala" \
         | awk -F \" '{print $2}' \
         )"
-    prior_version=$((current_version - 1))
+
+    # This function should only ever run on Travis PR builds where TRAVIS_PULL_REQUEST_BRANCH is set.
+    # https://stackoverflow.com/questions/229551/how-to-check-if-a-string-contains-a-substring-in-bash
+    if [ -z "${TRAVIS_PULL_REQUEST_BRANCH##*_hotfix}" ]; then
```

Comment:
I personally think an `egrep` with a real regex for `[0-9]+_hotfix` would be 
- (a) better (eg don't hit branches like `test_hotfix_logic`) and 
- (b) maybe even easier to read (I'd never seen this bash-ism for substrings before)

Reply:
discussed facially, when in Rome and all that

Reply:
Actually I think I'll implement that suggestion after all. The looseness of the substring match was bugging me but Bash regex matching is too quirky for my tastes.